apps/web/app/(app)/environments/[environmentId]/components/TopControlBar.tsx

```diff
@@ -1,34 +1,119 @@
+  organizations: TOrganization[];
+  project: TProject;
+  projects: TProject[];
+  isMultiOrgEnabled: boolean;
+  organizationProjectsLimit: number;
+  isFormbricksCloud: boolean;
+  isLicenseActive: boolean;
+  isOwnerOrManager: boolean;
+  isAccessControlAllowed: boolean;
   membershipRole?: TOrganizationRole;
   projectPermission: TTeamPermission | null;
 }
 
 export const TopControlBar = ({
   environment,
   environments,
+  organization,
+  organizations,
+  project,
+  projects,
+  isMultiOrgEnabled,
+  organizationProjectsLimit,
+  isFormbricksCloud,
+  isLicenseActive,
+  isOwnerOrManager,
+  isAccessControlAllowed,
   membershipRole,
   projectPermission,
 }: SideBarProps) => {
+  const { t } = useTranslate();
+  const router = useRouter();
+
+  const { isMember, isBilling } = getAccessFlags(membershipRole);
+  const { hasReadAccess } = getTeamPermissionFlags(projectPermission);
+  const isReadOnly = isMember && hasReadAccess;
+
   return (
     <div
-      className="flex h-14 w-full items-center justify-end bg-slate-50 px-6"
+      className="flex h-14 w-full items-center justify-between bg-slate-50 px-6"
       data-testid="fb__global-top-control-bar">
-      <div className="shadow-xs z-10">
-        <div className="flex w-fit items-center space-x-2 py-2">
-          <TopControlButtons
-            environment={environment}
-            environments={environments}
-            membershipRole={membershipRole}
-            projectPermission={projectPermission}
-          />
-        </div>
+      <div className="flex items-center">
+        <ProjectAndOrgSwitch
+          currentEnvironment={environment}
+          environments={environments}
+          currentOrganization={organization}
+          organizations={organizations}
+          currentProject={project}
+          projects={projects}
+          isMultiOrgEnabled={isMultiOrgEnabled}
+          organizationProjectsLimit={organizationProjectsLimit}
+          isFormbricksCloud={isFormbricksCloud}
+          isLicenseActive={isLicenseActive}
+          isOwnerOrManager={isOwnerOrManager}
+          isMember={isMember}
+          isAccessControlAllowed={isAccessControlAllowed}
+        />
+      </div>
+      <div className="z-50 flex items-center space-x-2">
+        <TooltipRenderer tooltipContent={t("common.share_feedback")}>
+          <Button variant="ghost" size="icon" className="h-fit w-fit bg-slate-50 p-1" asChild>
+            <Link
+              href="https://github.com/formbricks/formbricks/issues"
+              target="_blank"
+              rel="noopener noreferrer">
+              <BugIcon />
+            </Link>
+          </Button>
+        </TooltipRenderer>
+
+        <TooltipRenderer tooltipContent={t("common.account")}>
+          <Button
+            variant="ghost"
+            size="icon"
+            className="h-fit w-fit bg-slate-50 p-1"
+            onClick={() => {
+              router.push(`/environments/${environment.id}/settings/profile`);
```

Comment:
We should have a `<Link />` as the `<Button />` child to do the page linking.
https://ui.shadcn.com/docs/components/button

Then the component can also be a server-component which makes rendering and data handling more efficient.